src/stateToHTML.js

```diff
@@ -217,7 +217,10 @@ class MarkupGenerator {
       this.currentBlock += 1;
       return;
     }
-    this.writeStartTag(blockType, customRenderer, block.data);
+
+    if (block.text.trim() !== '' || isEmptyBlock(blockType)) {
+      this.writeStartTag(blockType, customRenderer, block.data);
+    }
 
     if (!isEmptyBlock(blockType)) {
       this.output.push(this.renderBlockContent(block));
```

Comment:
The block content is rendered either way (in the renderBlockContent, the <br> is rendered. So it needs to be rendered either way)